options/default.xml

```diff
@@ -599,17 +599,17 @@ QLineEdit#le_status_text {
                 <chat_share_files type="QKeySequence"></chat_share_files>
                 <chat_templates type="QKeySequence"></chat_templates>
                 <chat_voice type="QKeySequence"></chat_voice>
-                <gchat_info type="QKeySequence"></gchat_info>
                 <gchat_clear type="QKeySequence"></gchat_clear>
                 <gchat_configure type="QKeySequence"></gchat_configure>
                 <gchat_find type="QKeySequence"></gchat_find>
                 <gchat_html_text type="QKeySequence"></gchat_html_text>
                 <gchat_icon type="QKeySequence"></gchat_icon>
+                <gchat_infos type="QKeySequence"></gchat_infos>
```

Comment:
infos? From which word is it?

Reply:
yes, there are informations not only one information and there is gchat_info already ;)